diff -urN aws-c-mqtt-0.8.0-orig/CMakeLists.txt aws-c-mqtt-0.8.0/CMakeLists.txt
--- aws-c-mqtt-0.8.0-orig/CMakeLists.txt	2022-12-02 10:04:18.903047454 -0600
+++ aws-c-mqtt-0.8.0/CMakeLists.txt	2022-12-02 10:06:04.680196637 -0600
@@ -110,7 +110,7 @@
 endif()
 
 install(EXPORT "${PROJECT_NAME}-targets"
-        DESTINATION "${LIBRARY_DIRECTORY}/${PROJECT_NAME}/cmake/${TARGET_DIR}"
+        DESTINATION "${LIBRARY_DIRECTORY}/cmake/${PROJECT_NAME}/${TARGET_DIR}"
         NAMESPACE AWS::
         COMPONENT Development)
 
@@ -119,7 +119,7 @@
         @ONLY)
 
 install(FILES "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}-config.cmake"
-        DESTINATION "${LIBRARY_DIRECTORY}/${PROJECT_NAME}/cmake/"
+        DESTINATION "${LIBRARY_DIRECTORY}/cmake/${PROJECT_NAME}/"
         COMPONENT Development)
 
 include(CTest)
